admin_solutions/authentication.adoc

```diff
@@ -5,10 +5,303 @@ toc::[]
 {product-title} supports many different authentication methods, as defined in
 xref:../install_config/configuring_authentication.adoc#install-config-configuring-authentication[Configuring Authentication]:
 
+- xref:basic-auth-remote[Basic Authentication (Remote)]
+- xref:request-header-auth[Request Header]
 - xref:keystone-auth[Keystone]
 - xref:ldap-auth[LDAP]
 - xref:github-auth[GitHub]
 
+[[basic-auth-remote]]
+== Basic Authentication (Remote)
+
+[[BasicAuthPasswordIdentityProvider]]
+
+Basic Authentication is a generic backend integration mechanism that allows
+users to log in to {product-title} with credentials stored on a remote identity
+provider.
+
+[CAUTION]
+====
+Basic Authentication is not a secure exchange in principle; it needs to be
+combined with encryption or certificate authentication to prevent potential
+snooping of the user ID and password, and to prevent man-in-the-middle attacks.
```

Comment:
I understand what this is saying, but I would not lead with "Basic Authentication is not a secure exchange in principle". I would rephrase as "Basic Authentication should use a HTTPS connection to the remote server in order to prevent potential snooping of the user ID and password, and to prevent man-in-the-middle attacks."